lib/puppet/provider/jail/pyiocage.rb

```diff
@@ -6,28 +6,43 @@
   defaultfor kernel: :freebsd
 
   # this is used for further confinement
-  commands iocage: '/usr/local/bin/iocage'
+  commands pyiocage: '/usr/local/bin/iocage'
 
-  def self.pyiocage(*args)
+  def self.iocage(*args)
     cmd = ['/usr/local/bin/iocage', args].flatten.join(' ')
-    execute(cmd, override_locale: false)
+    execute(cmd, override_locale: false, failonfail: true)
```

Comment:
the last issue i'm struggling with are execution failures here.
`failonfail` doesn't seem to propagate the (stderr) error from `execute`